src/Runner.Common/JobServerQueue.cs

```diff
@@ -275,7 +275,6 @@ private async Task ProcessWebConsoleLinesQueueAsync(bool runOnce = false)
                         {
                             Trace.Info($"Skip {batchedLines.Count - 2} batches web console lines for last run");
                             batchedLines = batchedLines.TakeLast(2).ToList();
-                            batchedLines[0].Insert(0, "...");
```

Comment:
?

Reply:
I'm curious about this particular change as well. It doesn't seem part of the original port.

Reply:
White space diff problem?
https://github.com/actions/runner/pull/663/files/69721ef14400959170588d9ee82f746e9d0af84c?diff=split&w=1
how does this look @ericsciple  @juliobbv

Reply:
I don't think this was intentional, I will remove it, this is probably for cases where we skipped lines, so we send `...` for signalR

Reply:
oh it is intentional, it is also gone in devops https://github.com/microsoft/azure-pipelines-agent/blob/master/src/Microsoft.VisualStudio.Services.Agent/JobServerQueue.cs#L283